z3/src/ast.rs

```diff
@@ -2317,6 +2317,7 @@ pub fn exists_const<'ctx>(
 /// let f_f_3: ast::Int = f.apply(&[&f.apply(&[&ast::Int::from_u64(&ctx, 3)])]).try_into().unwrap();
 /// assert_eq!(3, model.eval(&f_f_3, true).unwrap().as_u64().unwrap());
 /// ```
+#[expect(clippy::too_many_arguments)]
```

Comment:
This is basically setting an MSRV of 1.81 on the crate. I'm probably fine with that.

Reply:
This is a good point; I forgot this was introduced this recently. I can change to `allow` if you prefer.

Reply:
I am fine with it as is and we see if anyone comments. There isn’t much point for a crate like this one to worry about older compilers.

Reply:
I went ahead and reverted this to `allow` as the package itself isn't currently specifying an MSRV and I don't want to break anyone's workflow without reason.